packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -120,144 +120,144 @@ const ClinicalResultBloodPressure = (props) => {
   let clinicalResultBloodPressureDisplay = null;
 
   if (hasResultError) {
-    clinicalResultBloodPressureDisplay = <ResultError />;
-  } else if (hasResultNoData) {
-    clinicalResultBloodPressureDisplay = <NoData />;
-  } else {
-    const compareUnits = {};
-    const compareStatusIsInError = {};
+    return <ResultError />;
+  }
+  if (hasResultNoData) {
+    return <NoData />;
+  } 
+  const compareUnits = {};
+  const compareStatusIsInError = {};
 
```

Comment:
I think it'd be work creating two new variables for systolic & diastolic with fallback to {} so we can remove all the extra `systolic && systolic.{key}` checks in this file to ensure systolic is not undefined 
```
const systolicResult = systolic || {};
const diastolicResult = diastolic || {};
```

Reply:
Added in the object, https://github.com/cerner/terra-clinical/pull/648/commits/a037421a0680c7296aa23625adb73abd2787d211
this might change as it looks like i'm gonna retool this section in general.